code/modules/atmospherics/components/unary/vent_pump.dm

```diff
@@ -554,6 +557,19 @@
 	. = ..()
 	air_contents.volume = ATMOS_DEFAULT_VOLUME_PUMP + 500 //meant to match air injector
 
+/obj/machinery/atmospherics/unary/vent_pump
+	
+/obj/machinery/atmospherics/unary/vent_pump/power_change()
+	. = ..()
+	if(.)
+		update_sound()
+
+/obj/machinery/atmospherics/unary/vent_pump/proc/update_sound()
+	if(can_pump())
+		sound_token = GLOB.sound_player.PlayLoopingSound(src, "[type]_[x]_[y]_[z]", 'sound/machines/vent_hum.ogg', 15, range = 14, falloff = 4)
```

Comment:
Due to BYOND-limitations we can only have 1024 unique sound sources and vents would take up a big part of those
Generally objects playing the same sound should all utilize the same sound id

Reply:
Changed it to only enforce uniqueness inside zlevel. As far as people can hear, they're all unique.